test/integration-test/CodeGen/copy_stencil_benchmark.cpp

```diff
@@ -32,9 +32,12 @@ TEST(copy_stencil, test) {
   verif.fill(-1.0, out_gt, out_naive);
 
   gridtools::copy_stencil copy_gt(dom, in, out_gt);
+  verif.sync_storages(in);
```

Comment:
1) do we need to sync before the fill?
2) we should use consistent syntax: I would suggest to use verif.sync_storages() also for 1 argument

Reply:
1. it is done within the fill
2. yes, we are already doing that, the line below with `in.sync()` shouldnt be there anymore I removed it